### PR TITLE
Add versioning property

### PR DESCRIPTION
**Pull request by tsaiggo**
# Choose a PR Template

Switch to "Preview" on this description then select one of the choices below.

<a href="?expand=1&template=data_plane_template.md">Click here</a> to open a PR for a Data Plane API.

<a href="?expand=1&template=control_plane_template.md">Click here</a> to open a PR for a Control Plane (ARM) API.

### CODE REVIEW

**Review by ruowan — APPROVED** [review]

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json [hunk 1]
```diff
@@ -26,6 +26,10 @@
           "type": "string",
           "description": "Type of linked service."
         },
+        "version": {
+          "type": "string",
+          "description": "Version of the linked service."
+        },
```
Comment (qiaozha) [inline specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json:32]: @tsaiggo could you check if the version in HDInsightOnDemandLinkedService is still necessary?

Comment (ruowan) [reply specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json:32]: Yes. It's necessary. This HDInsightOnDemandLinkedService is owned by HDInsight team. They have their own business logic. 

Btw, any impact for the downstream SDK?

**Comment by ruowan** [comment]
Added back the labels which was removed by the bot mistakely.

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json (hunk 1 above)
Comment (qiaozha) [reply specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json:32]: It's causing duplicate version properties for the generated code from versionTypePropertiesVersion, and codegen now rename the original version in `HDInsightOnDemandLinkedService` as `versionTypePropertiesVersion`
```
- Interface HDInsightOnDemandLinkedService has a new required parameter versionTypePropertiesVersion
```

Comment (ruowan) [reply specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json:32]: Which SDK and will customer be blocked?

**Pull request merged**